diagnostics/app/model/diagnostics/analytics/Metric.scala

```diff
@@ -13,11 +13,15 @@ object Metric extends Logging {
 
   lazy val namespace = "Diagnostics"
 
+  // TODO delete bounce-test-present when this goes
+  import conf.Switches.NoBounceIndicator
+
   lazy val metrics = Map(
 
     // page views
     ("pv", CountMetric("kpis-page-views")),            // raw page views - simple <img> in body, no javascript involved
     ("pva", CountMetric("kpis-analytics-page-views")), // page view fires after analytics
+    ("bounce-test-present", CountMetric("bounce-test-present")),
```

Comment:
I'd have called it user-bounced-prematurely or something, as the bounce test is always present when the switch is on so the name is confusing